### PR TITLE
[ELI5] change sign in to create account

### PR DESCRIPTION
**Pull request by mozzius**
Also ALF'd some buttons

Screenshot 2024-12-03 at 21 29 45

Screenshot 2024-12-03 at 21 29 58

### CODE REVIEW

**Review by emilyliu7321 — APPROVED** [review]

File: src/view/shell/NavSignupCard.tsx [hunk 1]
```diff
@@ -44,12 +44,12 @@ let NavSignupCard = ({}: {}): React.ReactNode => {
...
         <Button
           onPress={showCreateAccount}
-          label={_(msg`Sign up`)}
+          label={_(msg`Create account`)}
           size="small"
           variant="solid"
           color="primary">
           <ButtonText>
-            <Trans>Sign up</Trans>
+            <Trans>Create account</Trans>
```
Comment (estrattonbailey) [inline src/view/shell/NavSignupCard.tsx:52]: Can we specify an `id` for Lingui so that this string doesn't change in other langs?

**Review by estrattonbailey — APPROVED** [review]
If we can save translators effort, should do that. But if we think this is more clear and can't avoid new translations, then that's a tradeoff we can take.
